app/build.gradle

```diff
@@ -261,13 +261,34 @@ dependencies {
     implementation 'com.google.firebase:firebase-crashlytics-ktx:17.3.1'
     implementation 'com.google.firebase:firebase-iid:21.0.1'
 
-    testImplementation "io.mockk:mockk:1.10.0"
-    testImplementation "junit:junit:4.13.1"
-    testImplementation "androidx.room:room-testing:2.2.6"
-    testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.3.9"
+    // Testing
```

Comment:
Undo all changes here unless it's 100% required.

Reply:
I added these dependencies since I wanted to tests out the other components like db but I started with the viewmodels

Reply:
You added things such as Hilt testing. Why did you add testing for Hilt? You also added a lot of Mockito even though we're using Mockk.

Unless you're actually using it in the PR, please remove it.